src/EmergencyDevice.ts

```diff
@@ -0,0 +1,71 @@
+import { t } from 'ttag';
+import SimpleSchema from 'simpl-schema';
+import './SimpleSchemaExtensions';
+import { Length, LengthSchema, quantityDefinition } from './Units';
+import { LocalizedStringSchema, LocalizedString } from './LocalizedString';
+
+/**
+ * Describes a EmergencyDevice unit provided at this place, for example an audibleFireAlarm
+ */
+export interface EmergencyDevice {
+  /**
+   * Type of the EmergencyDevice unit
+   */
+  type: 'evacChair' | 'alarm' | 'visualFireAlarm' | 'audibleFireAlarm';
+
+  /**
+   * Is the emergency device in audio
+   */
+  isAudio?: boolean;
+
+  /**
+   * Is the emergency device visual
+   */
+  isVisual?: boolean;
+
+  /**
+   * Is the emergency device visual
+   */
+  isAvailable?: boolean;
+}
+
+export const EmergencyDeviceSchema = new SimpleSchema({
+  type: {
+    type: String,
+    label: t`EmergencyDevice Type`,
```

Comment:
```suggestion
    label: t`Kind of emergency device`,
```

(These strings are meant to be displayed to non-developer users, too)

Reply:
Typical case of copy-pasting on my behalf ...